app/templates/cli-config.yml

```diff
@@ -29,9 +29,9 @@ container-port-map : "3000:3000"
 container-port-map-debug : "5858:5858"
 
 # The name for the dockerfile for the run container
-dockerfile-run : "Dockerfile"
+dockerfile-run : "docker-compose.yml"
```

Comment:
Only MEAN/MERN starters use compose files.  All other starters use dockerfiles.  So this file needs conditional logic, triggered by a flag passed in by the MEAN/MERN starter, requesting "enablement for multiple containers"